pkg/cli/validate.go

```diff
@@ -41,23 +47,84 @@ func runValidateCmd(cmd *cobra.Command, args []string) error {
 		Version:         "1",
 		JWKSEndpoint:    cfg.JWKSEndpoint,
 		CacheTimeout:    5 * time.Minute,
-		AllowBreakglass: flagAllowBreakglass,
+		AllowBreakglass: true,
 	})
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to create jvs client: %w", err)
 	}
 
-	_, err = jvsclient.ValidateJWT(flagToken)
+	if flagToken == "-" {
+		stat, _ := os.Stdin.Stat()
+		if (stat.Mode() & os.ModeCharDevice) == 0 {
+			// Read from pipe
```

Comment:
It might be shorter to read all of stdin:


```go
b, err := ioutil.ReadAll(io.LimitReader(stdin, 64*1_000))
```

Also, I would recommend making `os.Stdin` a private global variable. It makes overriding in tests easier.

Reply:
Used io.ReadAll since ioutil.ReadAll is deprecated.

Reply:
Sorry - old habits :)